webpack.config.js

```diff
@@ -71,6 +71,7 @@ webpackConfig.plugins.push(new ObjectRestSpreadPlugin());
 if (!isProduction) {
   webpackConfig.module = webpackConfig.module || {};
   webpackConfig.module.rules = webpackConfig.module.rules || [];
+  webpackConfig.externals = [/devtools\/shared\/l10n/];
```

Comment:
can we update `tranform-mc` to avoid the need for this. I think it is better to do all import logic in one place